classes/patreon_options.php

```diff
@@ -172,11 +154,21 @@ function patreon_plugin_setup_page(){
 											<th scope="row">
 												<strong>Enable strict oAuth</strong>
 												<br>
-												(Only connects users logged into WordPress)
+												<div class="patreon-options-info">If on, the plugin will only connect users who are already logged into your WordPress website. If off, new accounts will be created automatically for users who are logging in for the first time via Patreon. Recommended: off</div>
 											</th>
 											<td>
 												<input type="checkbox" name="patreon-enable-strict-oauth" value="1"<?php checked( get_option('patreon-enable-strict-oauth', false) ); ?> />
 											</td>
+                                        </tr>
+                                        <tr valign="top">
+											<th scope="row">
+												<strong>Enable image locking features</strong>
+												<br>
+												<div class="patreon-options-info">If on, you will be able to lock your images and provide patron only images anywhere in your posts like webcomics or visual content. If you aren't using image locking or having complications due to your web host infrastructure, you can keep this feature off. Whenever you turn this feature on or off, you should visit 'Permalinks' settings in your WordPress site and save your permalinks just once by clicking 'Save'. Otherwise your images may appear broken.</div>
+											</th>
+											<td>
+												<input type="checkbox" name="patreon-enable-file-locking" value="1"<?php checked( get_option('patreon-enable-file-locking', false) ); ?> />
+											</td>
```

Comment:
The HTML here is pretty messy, let's clean up

Reply:
The inline PHP bit?